docs/en/parametri_scope_claims.rst

```diff
@@ -24,13 +24,13 @@ In case of using the **scope** parameter, the following values are supported:
 The attributes requested by the parameter **scope** are available both in the ID Token and in the response to the Userinfo Endpoint.
 
 .. note::
-   The parameter **scope** MAY contain one or more values, with single spaces as separators.
+   The parameter **scope** MAY contain one or more values, with single spaces as separators. For example, using both *profile* and *email* in the **scope** parameter gives the Minimum Dataset eIDAS and the email.
 
 In case of requests of single user-attributes or specific combinations of them, the RP MAY use the parameter **claims**. 
 For the definition of the parameter **claims** and its usage modes for requesting the user attributes, please refer to `OpenID.Core#ClaimsParameter`_. 
 
 .. warning::
-    - In the object *id_token* of the parameter **claims**, it is possible to request only the eIDAS Minimum  Dataset. The other user attributes MUST be requested in the object *userinfo* of the parameter **claims**.  
+    - In the object *id_token* of the parameter **claims**, it is possible to request only the eIDAS Minimum Dataset. The other user attributes MUST be requested in the object *userinfo* of the parameter **claims**. Moreover, the user attributes requested in the *id_token* object are also available at the *userinfo endpoint*.  
```

Comment:
@agcolella in SPID OIDC guidelines, none user attributes are expected. It's not possible to request user attributes in the id_token, so neither the attributes related to the eIDAS Minimum Dataset

Reply:
Done here
https://github.com/italia/spid-cie-oidc-docs/pull/51/commits/0bc962d37e05b78c51eaa0206b901d7419151d06